blog/2023-10-29-ratify-1.0.0.mdx

```diff
@@ -0,0 +1,80 @@
+Ratify uses extensible plugin architecture to enable extensibility and interoperability to support both the existing and new emerging needs of the artifact verification. Developers can easily [create a new plugin](https://ratify.dev/docs/1.0/quickstarts/creating-plugins) and integrate into Ratify ecosystem. See [Framework overview](https://ratify.dev/docs/1.0/reference/ratify-framework-overview) for details. 
+
+![](/img/ratify-core-principals.png)
+
+## Observability
+
+Observability is the process of gaining insight into the behavior and performance of applications running on Kubernetes, in order to monitor, troubleshoot, and identify issues more effectively. 
+Ratify introduces standardized logging component and metrics exporter for Prometheus in this release so that users can gather and visualize the metrics via the popular monitoring and logging stack, like Prometheus, Grafana, and EFK.
+
+With the rich metrics and verbose logs, it can help ensure the stability and performance of Kubernetes deployments and Ratify system, reduce verification request latency, system downtime and improve security posture.
+All available metrics are listed in this [doc](https://ratify.dev/docs/1.0/reference/instrumentation#metrics-supported). Refer to this [guide](https://ratify.dev/docs/1.0/reference/instrumentation#prometheus-and-grafana-setup) to build your own visualized dashboard for Ratify with Prometheus and Grafana. 
+
+![](/img/ratify-grafana.png)
+
+## Support dynamic configuration
+
+Ratify supports updating its common configuration through Kubernetes CRDs using kubectl commands, delivers a Kubernetes-native experience for users. The configuration takes effect dynamically so users no need to restart the running Ratify. There are four CRDs including Certificate Store, Policy, Store, and Verifier. You can see [CRD reference docs](https://ratify.dev/docs/1.0/reference/crds/certificate-stores) for details.
+
+## Ready for large-scale scenarios
+
+In order to provide security and reliability guarantees for production users in a large-scale deployment scenario, Ratify maintainers finished the load performance testing and published a performance report with detailed performance metrics. It demonstrates Ratify is ready for large-scale deployment scenario. See [performance at scale](https://ratify.dev/docs/1.0/reference/performance/) for details.
+
+## High Availability (Experimental)
+
+For higher performance and availability requirements, Ratify can be set to run with multiple replicas and a shared state store.
+Ratify supports High Availability (HA) using a distributed cache with Dapr and Redis under the hood. HA is an experimental feature in Ratify v1.0.0. Experimental features are intended for testing and evaluation purposes only and should not be used in production environments. See [Install Ratify for High Availability](https://ratify.dev/docs/1.0/quickstarts/ratify-high-availability) to set up an HA cluster.
```

Comment:
We have a known [issue](https://github.com/deislabs/ratify/issues/1117) on distributed cache, wonder if we should call it out?

Reply:
The known issues should be stated in user manual instead of blog post because blog post is for marketing purposes.